cmake/OpenCVCompilerOptimizations.cmake

```diff
@@ -390,26 +415,13 @@ elseif(PPC64LE)
   set(CPU_BASELINE "VSX" CACHE STRING "${HELP_CPU_BASELINE}")
 
 elseif(RISCV)
-  if(NOT DEFINED PLATFORM_STR)
-    set(PLATFORM_STR "rv64gc")
-  endif()
 
-  ocv_update(CPU_KNOWN_OPTIMIZATIONS "RVV;FP16;RVV_ZVFH")
```

Comment:
We need to keep FP16 and RVV_ZVFH in the list of known optimizations (line 55, line 397) as well as TEST_FILE, IMPLIES, FLAGS_ON variables (lines 399-407).

I'll adapt flags-riscv64.cmake accordingly after the merge.

Reply:
@mshabunin Fixed.

Reply:
Done.